### ISSUE TITLE
[NPC][Northrend]Cavedweller Worg/Infesting Jormungar

### ISSUE THREAD

**Issue by ImprezaWoW** [dalaranwow/dalaran-wow#3895]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**:
As of 28-11 (today's or yesterday night's maintenance, probably), Cavedweller Worgs, most notably needed for the quest [Emergency Measures](http://www.wowhead.com/quest=13000/emergency-measures), seem to be MIA, and most of the Infesting Jormungars, most notably needed for [Cave Medicine](http://www.wowhead.com/quest=13055/cave-medicine), are missing, to be precise only 2 out of 16 are spawned, and the ones which are present can not be targeted, you must go to their aggro range and be attacked by them so you would be able to target them.
They are both located in the same cave.
**Expected behaviour**: Tell us what should happen instead
The 20 worg spawns should be visible to players, so should the other 14/16 jormungars, besides making them targettable.
**Steps to reproduce the problem**:

1. Go to the cave in the Storm Peaks where they should be spawning.
2. In the tunnel leading to the end room, there are no Cavedweller Worgs, nor Infesting Jormungars.
3. In the end room, there are only 2 Jormungars spawned.

**Include proofs for this behaviour**

https://youtu.be/3-1cbxoCWQw
Both creature behaviour can be seen in this video.
**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

[Cavedweller Worg](http://www.wowhead.com/npc=30164/cavedweller-worg) NPC ID 30164
[Infesting Jormungar](http://www.wowhead.com/npc=30148/infesting-jormungar) NPC ID 30148
**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
```sql
DELETE FROM `creature` WHERE `id ` IN (30148, 30164);
INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `VerifiedBuild`) VALUES 
(43115, 30148, 571, 0, 0, 1, 1, 26263, 0, 7580.26, -1470.25, 974.674, 3.75728, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43114, 30148, 571, 0, 0, 1, 1, 26263, 0, 7905.25, -1579.47, 915.558, 5.92105, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43113, 30148, 571, 0, 0, 1, 1, 26263, 0, 7894.05, -1547.91, 920.607, 4.99429, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43112, 30148, 571, 0, 0, 1, 1, 26263, 0, 7868.22, -1521.16, 925.599, 5.97211, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43111, 30148, 571, 0, 0, 1, 1, 26263, 0, 7820.32, -1549.61, 935.184, 3.57664, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43110, 30148, 571, 0, 0, 1, 1, 26263, 0, 7791.33, -1564.27, 941.703, 1.46392, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43109, 30148, 571, 0, 0, 1, 1, 26263, 0, 7737.69, -1625.75, 950.588, 1.43251, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43108, 30148, 571, 0, 0, 1, 1, 26263, 0, 7710.24, -1635.46, 952.952, 2.84622, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43107, 30148, 571, 0, 0, 1, 1, 26263, 0, 7631.37, -1591.72, 972.049, 1.12227, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43106, 30148, 571, 0, 0, 1, 1, 26263, 0, 7645.64, -1533.16, 973.012, 4.99036, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43105, 30148, 571, 0, 0, 1, 1, 26263, 0, 7615.32, -1550.68, 972.753, 4.02432, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43104, 30148, 571, 0, 0, 1, 1, 26263, 0, 7606.6, -1522.91, 974.029, 3.7887, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43103, 30148, 571, 0, 0, 1, 1, 26263, 0, 7920.24, -1625.21, 910.852, -1.9338, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43102, 30148, 571, 0, 0, 1, 1, 26263, 0, 7957.04, -1610.1, 908.131, 0.20944, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43101, 30148, 571, 0, 0, 1, 1, 26263, 0, 7996.98, -1608.36, 908.772, 4.2586, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(43100, 30148, 571, 0, 0, 1, 1, 26263, 0, 7990.3, -1559.1, 904.997, 2.82743, 300, 5, 0, 12600, 0, 1, 0, 0, 0, 0),
(4109127, 30164, 571, 0, 0, 1, 1, 22042, 0, 7575.7, -1474.36, 977.123, 0.768835, 300, 0, 0, 8820, 0, 0, 0, 0, 0, 0),
(4109126, 30164, 571, 0, 0, 1, 1, 22611, 0, 7914.54, -1578.14, 914.215, 3.41956, 300, 0, 0, 8820, 0, 0, 0, 0, 0, 0),
(4109125, 30164, 571, 0, 0, 1, 1, 22042, 0, 7909.72, -1525.18, 918.712, 3.31747, 300, 0, 0, 8820, 0, 0, 0, 0, 0, 0),
(4109124, 30164, 571, 0, 0, 1, 1, 22611, 0, 7888.94, -1522.08, 921.393, 3.30176, 300, 0, 0, 8820, 0, 0, 0, 0, 0, 0),
(4109123, 30164, 571, 0, 0, 1, 1, 22042, 0, 7846.2, -1518.94, 929.342, 6.12919, 300, 0, 0, 8523, 0, 0, 0, 0, 0, 0),
(4109122, 30164, 571, 0, 0, 1, 1, 22611, 0, 7742.93, -1605.62, 947.645, 4.55446, 300, 0, 0, 8820, 0, 0, 0, 0, 0, 0),
(4109121, 30164, 571, 0, 0, 1, 1, 22042, 0, 7603.8, -1566.06, 972.708, 0.635327, 300, 0, 0, 8820, 0, 0, 0, 0, 0, 0),
(4109120, 30164, 571, 0, 0, 1, 1, 22611, 0, 7601.32, -1527.01, 974.217, 0.427199, 300, 0, 0, 8820, 0, 0, 0, 0, 0, 0),
(4109119, 30164, 571, 0, 0, 1, 1, 22611, 0, 7923.33, -1629.44, 911.088, 2.20218, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109118, 30164, 571, 0, 0, 1, 1, 22042, 0, 7924.59, -1628.15, 910.836, 2.54659, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109117, 30164, 571, 0, 0, 1, 1, 22611, 0, 7920.12, -1630.42, 911.431, 1.54768, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109115, 30164, 571, 0, 0, 1, 1, 22611, 0, 7989.47, -1554.61, 905.15, 4.95674, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109116, 30164, 571, 0, 0, 1, 1, 22042, 0, 7918.38, -1630.09, 911.414, 4.17355, 300, 5, 0, 8522, 0, 1, 0, 0, 0, 0),
(4109114, 30164, 571, 0, 0, 1, 1, 22611, 0, 7987.4, -1561.01, 905.209, 0.506145, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109113, 30164, 571, 0, 0, 1, 1, 22611, 0, 7951.54, -1607.11, 908.721, 5.74213, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109112, 30164, 571, 0, 0, 1, 1, 22042, 0, 7959.13, -1606.12, 907.662, 4.36332, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109111, 30164, 571, 0, 0, 1, 1, 22611, 0, 7993.44, -1610.69, 908.332, 0.418879, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109110, 30164, 571, 0, 0, 1, 1, 22042, 0, 7995.35, -1605.57, 908.536, 5.18363, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109109, 30164, 571, 0, 0, 1, 1, 22611, 0, 8029.42, -1596.67, 903.312, 3.03687, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0),
(4109108, 30164, 571, 0, 0, 1, 1, 22042, 0, 8029.98, -1589.47, 903.184, 3.36848, 300, 0, 0, 8522, 0, 0, 0, 0, 0, 0);
```
DB/Creature: Respawn Cavedweller Worg/Infesting Jormungar 
* closes https://github.com/dalaranwow/dalaran-wow/issues/3895
* commit: 66edc2ffca20edfda61d2e24c1fd2913a0f3d7ba

This works fine on PTR @Lothloryen I Think ypu just need to respawn the creature from ptr, which i posted above. Thanks

**Comment by Zibathy** [comment]
Now there are 4/16 spawned on Kirin Tor.

**Comment by TeridanDW** [comment]
Hello, this cave was completely reworked, the gameobjects in #3588 and the creatures in #4360
@Rushor maybe do not apply this commit and take the gameobjects from #3588 and creature fixes from #4360 instead?
Reopening as decision must be taken before these 3 commits are applied on ptr or it will be a mess.

**Comment by Rushor** [comment]
Yes i already removed this commit/spawns and replaced it with the mentioned fixes from https://github.com/dalaranwow/dalaran-wow/issues/4360.

So i will close this issue and mark it as duplicate because everything was reworked in https://github.com/dalaranwow/dalaran-wow/issues/4360
